From 27a6f61967a3bf273a6b454aa476a568d1b151af Mon Sep 17 00:00:00 2001
From: darkobas <darkobas@gmail.com>
Date: Fri, 21 Oct 2016 19:55:57 +0200
Subject: [PATCH] remove project path

---
 Android.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Android.mk b/Android.mk
index 11d6f84..0bc2870 100644
--- a/Android.mk
+++ b/Android.mk
@@ -1,4 +1,4 @@
-ifeq ($(call my-dir),$(call project-path-for,qcom-audio))
+ifeq ($(TARGET_QCOM_AUDIO_VARIANT),caf-msm8974)
 
 ifneq ($(filter mpq8092 msm8960 msm8226 msm8x26 msm8610 msm8974 msm8x74 apq8084,$(TARGET_BOARD_PLATFORM)),)
 
-- 
2.11.0

