From 649992cea9c6a32a1b0f5b8e6432e95a1e2fdd1a Mon Sep 17 00:00:00 2001
From: sub77 <sub77@ymail.com>
Date: Mon, 29 Oct 2018 12:15:01 +0000
Subject: [PATCH] include: Add support for O_TMPFILE

---
 include/asm-generic/fcntl.h | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/include/asm-generic/fcntl.h b/include/asm-generic/fcntl.h
index 9e5b0356e2b..1f3f063f434 100644
--- a/include/asm-generic/fcntl.h
+++ b/include/asm-generic/fcntl.h
@@ -84,6 +84,13 @@
 #define O_PATH		010000000
 #endif
 
+#ifndef __O_TMPFILE
+#define __O_TMPFILE	020000000
+#endif
+
+#define O_TMPFILE (__O_TMPFILE | O_DIRECTORY)
+#define O_TMPFILE_MASK (__O_TMPFILE | O_DIRECTORY | O_CREAT)
+
 #ifndef O_NDELAY
 #define O_NDELAY	O_NONBLOCK
 #endif
-- 
2.17.2

