From 741c3fc8a81b9425a7a4b8b5702007c9b5b47e01 Mon Sep 17 00:00:00 2001
From: sub77 <darfpunk@gmx.de>
Date: Wed, 21 Dec 2016 14:50:26 +0000
Subject: [PATCH] TW_DEVICE_SPECIFIC_VERSION

Signed-off-by: sub77 <darfpunk@gmx.de>
---
 Android.mk  | 3 +++
 variables.h | 4 ++++
 2 files changed, 7 insertions(+)

diff --git a/Android.mk b/Android.mk
index 582b710..bcaf7e1 100644
--- a/Android.mk
+++ b/Android.mk
@@ -352,6 +352,9 @@ ifneq ($(TW_DEFAULT_LANGUAGE),)
 else
     LOCAL_CFLAGS += -DTW_DEFAULT_LANGUAGE=en
 endif
+ifneq ($(TW_DEVICE_SPECIFIC_VERSION),)
+    LOCAL_CFLAGS += -DTW_DEVICE_SPECIFIC_VERSION=\"$(TW_DEVICE_SPECIFIC_VERSION)\"
+endif
 
 LOCAL_ADDITIONAL_DEPENDENCIES += \
     dump_image \
diff --git a/variables.h b/variables.h
index 9cdbfb6..b70ae35 100644
--- a/variables.h
+++ b/variables.h
@@ -17,7 +17,11 @@
 #ifndef _VARIABLES_HEADER_
 #define _VARIABLES_HEADER_
 
+#ifdef TW_DEVICE_SPECIFIC_VERSION
+#define TW_VERSION_STR TW_DEVICE_SPECIFIC_VERSION
+#else
 #define TW_VERSION_STR              "3.0.3-0"
+#endif
 
 #define TW_USE_COMPRESSION_VAR      "tw_use_compression"
 #define TW_FILENAME                 "tw_filename"
-- 
2.1.4

